### PR TITLE
Add Movies module

### PR DESCRIPTION
**Pull request by devrrior**
Hi!
I added the movies module. I developed two endpoints for this module (list movies and get movie by id). Also, I added OpenApi documentation and tests for routes and utils.

Issue #206

### CODE REVIEW

File: modules/movies/api/movies-routes.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+     *    responses:
+     *      '200':
+     *        description: OK
+     *        schema:
+     *          type: array
+     *          items:
+     *            $ref: '#/definitions/MockMovie'
+     */
+    app.get('/movies', (req, res) => {
+        const { qty = 10 } = req.query;
```
Comment (ageddesi) [inline modules/movies/api/movies-routes.ts:28]: Hi we have a util for getting the qty from the route, getCountryNameFromRequest(req);
also your route does not follow our pattern eg. '/address/:country?/:qty?'

**Review by ageddesi — CHANGES_REQUESTED** [review]
Hey @devrrior Just 1 change and is all good to be Merged.

File: modules/movies/api/movies-routes.ts (hunk 1 above)
Comment (devrrior) [reply modules/movies/api/movies-routes.ts]: Hi, sure. The changes have been made

**Comment by ageddesi** [comment]
hi @devrrior Your movie tests seem to be failing. :(

**Comment by devrrior** [comment]
> hi @devrrior Your movie tests seem to be failing. :(

It's strange when I run the tests that say everything is ok
image

**Comment by devrrior** [comment]
Hi @ageddesi, I've been running the tests in multiple times and I haven't found any problem. Jest says every tests have been passed.

image

**Comment by ageddesi** [comment]
Hi @devrrior I have just tried re-running and it failed again, The build issue is something that was fixed recently in dev. Do you have the latest code in this branch?

**Review by ageddesi — APPROVED** [review]

**Comment by ageddesi** [comment]
Hey @devrrior, I have updated the branch and made the tests pass 👍 Happy hacktoberfest and thank you for all the hard work on this.

**Pull request merged**